extensions/azurePublish-new/src/components/provisioningWizards/steps/ImportActionContentStep.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+import formatMessage from 'format-message';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+
+import { Content, Details, LearnMoreLink, ResourceTitle, Summary, Title } from '../../ChooseProvisionAction';
+
+export const ImportActionContentStep = () => {
+  return (
+    <Content>
+      <Title>{formatMessage('Use existing resources')}</Title>
+      <Summary>
+        <p>
```

Comment:
why paragraphs?

Reply:
In the existing UI, each instruction is shown in the next line. using p to get the same styling.